src/css/main.less

```diff
@@ -472,20 +473,22 @@ fieldset[disabled] .btn-info.active {
 }
 
 .nav-tabs > li > a {
-  color: #8e929b;
+  color: @navbar-inactive-color;
   border-top: none;
   border-left: none;
   border-right: none;
   border-radius: 0;
+  min-width: @navbar-min-width;
```

Comment:
I don't mind the existence of a base unit, but I'd prefer that component dimensions weren't directly related to padding in a way that could break layout when we increase or decrease spacing.